k8s/agent-scaler/Dockerfile

```diff
@@ -12,8 +12,14 @@ RUN yum repolist && \
         gettext \
         rh-python38 \
         rh-python38-python-pip && \
-    curl -Ls https://dl.k8s.io/release/v1.20.0/bin/linux/amd64/kubectl \
-         -o /usr/local/bin/kubectl && \
+    yum -y clean all --enablerepo='*'
+RUN set -eux; \
```

Comment:
qq. Why do we create a new layer?

Reply:
Returned to same layer, rebuild here
```
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                                               0.0s
 => => transferring dockerfile: 1.57kB                                                                                                                                                                                                                                                                             0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                                                    0.0s
 => [internal] load metadata for docker.io/library/centos:7                                                                                                                                                                                                                                                        1.4s
 => CACHED [1/8] FROM docker.io/library/centos:7@sha256:0f4ec88e21daf75124b8a9e5ca03c37a5e937e0e108a255d890492430789b60e                                                                                                                                                                                           0.0s
 => [internal] load build context                                                                                                                                                                                                                                                                                  0.0s
 => => transferring context: 31.51kB                                                                                                                                                                                                                                                                               0.0s
 => [2/8] RUN yum repolist &&     yum install -y         centos-release-scl-rh         epel-release         scl-utils         yum-utils &&     yum -y install         gettext         rh-python38         rh-python38-python-pip &&     yum -y clean all --enablerepo='*' &&     case $(uname -m) in              33.4s
 => [3/8] COPY agent-scaler.sh /tools/                                                                                                                                                                                                                                                                             0.0s
 => [4/8] COPY ensemble_count.py /tools/                                                                                                                                                                                                                                                                           0.0s
 => [5/8] COPY joshua_model.py /tools/                                                                                                                                                                                                                                                                             0.0s
 => [6/8] RUN chmod +x     /tools/agent-scaler.sh     /tools/ensemble_count.py     /tools/joshua_model.py                                                                                                                                                                                                          0.2s
 => [7/8] RUN curl -Ls https://www.foundationdb.org/downloads/6.2.29/linux/libfdb_c_6.2.29.so          -o /lib64/libfdb_c.so &&     chmod +x /lib64/libfdb_c.so                                                                                                                                                    5.8s 
 => [8/8] RUN source /opt/rh/rh-python38/enable &&     pip3 install foundationdb==6.2.10                                                                                                                                                                                                                           1.7s
 => exporting to image                                                                                                                                                                                                                                                                                             0.8s
 => => exporting layers                                                                                                                                                                                                                                                                                            0.8s
 => => writing image sha256:019de1f738962d483560d74bf7a725f64ba9ac8e885f7d926efdcfe61fafc5da                                                                                                                                                                                                                       0.0s
 => => naming to docker.io/foundationdb/agent-scaler:20210901154937-aeb5b56af0                                                                                                                                                                                                                                     0.0s
 => => naming to docker.io/foundationdb/agent-scaler:latest
```

Reply:
Also ran a local copy of image, used docker exec to get to bash and verified kubectl works